plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -41,6 +41,7 @@
     private final AutoCloseableCloser closer = AutoCloseableCloser.create();
 
     private State state = State.INITIAL;
+    private AmazonS3 s3Client;
```

Comment:
Should this be `s3Client.shutdown()` in `stop()`?

(pre-existing, let's fix in separate PR)

Reply:
As i've extracted it to become property it's fair to register it now in closer after it's created in start method.